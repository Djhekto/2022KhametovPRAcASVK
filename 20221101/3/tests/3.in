a, b = Grange(10, 10, 100000), Grange(1, 10, 1)
print(f"{len(a)=}, {len(b)=}")
print(f"{list(a)}({not(a)}), {list(b)}({not(b)})")
print(f"{a=}, {b=}")
print(f"{len(list(a))=}, {len(list(b))=}")
print(f"{a[5]=}, {a[21]=}")
a, b = a[1000:100000], a[1000:100000:2]
print(f"{a=}, {a[7]=}")
print(f"{b=}, {b[8]=}, {list(b)=}")