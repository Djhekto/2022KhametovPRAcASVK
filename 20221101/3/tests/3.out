len(a)=4, len(b)=0
[10, 100, 1000, 10000](False), [](True)
a=grange(10, 10, 100000), b=grange(1, 10, 1)
len(list(a))=4, len(list(b))=0
a[5]=1000000, a[21]=10000000000000000000000
a=grange(1000, 10, 100000), a[7]=10000000000
b=grange(1000, 100, 100000), b[8]=10000000000000000000, list(b)=[1000]
